src/arrayMethodSort.js

```diff
@@ -4,8 +4,23 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => a.toString() > b.toString(),
+  ) {
+    for (let i = 1; i < this.length; i++) {
+      const current = this[i];
+
+      let j = i - 1;
+
+      while (j >= 0 && compareFunction(this[j], current) > 0) {
```

Comment:
The comparison function should return a number less than 0, 0, or greater than 0. It is not guaranteed that it will return only -1, 0, or 1. Therefore, you should check if the result is less than 0, not just greater than 0.